packages/app/src/systems/Chains/fuel/utils/chain.ts

```diff
@@ -0,0 +1,5 @@
+import type { SupportedChain } from '../../types';
+
+export const isFuelChain = (chain: SupportedChain | undefined | null) => {
```

Comment:
```suggestion
export const isFuelChain = (chain?: SupportedChain | null) => {
```

would this be equivalent?